recherche/UtilisabiliteAout2020.md

```diff
@@ -0,0 +1,177 @@
+[English](https://github.com/cds-snc/covid-alert-documentation/blob/main/recherche/UsabilityAug2020.md)
```

Comment:
I think we can use relative path here too
`[English](recherche/UsabilityAug2020.md)`